sdk/communication/Azure.Communication.CallAutomation/src/Models/InterruptAudioAndAnnounceOptions.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Azure.Communication.CallAutomation
+{
+    /// <summary>
+    /// Options to be used in the Play operation.
+    /// </summary>
+    public class InterruptAudioAndAnnounceOptions
+    {
+        /// <summary>
+        /// PlaySource objects representing the sources to play.
+        /// </summary>
+        public IReadOnlyList<PlaySource> Announcement { get; }
```

Comment:
nit: plural announcements

Reply:
mmmmh I would argue that this is a single announcement potentially composed of different audio files. The semantic of the API is that you are making an announcement and that's why you are interrupting the current audio. 
Happy to hear what others think about this.